src/file-tester/lms_file_tester/__main__.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import logging
+import os
+import sys
+from typing import List, Union
+
+# This next line *should* have
+# `lms_file_tester.validators.directory_validation`, but the script does not run
+# from the command line if you have `lms_file_tester` there - you get a module
+# not found error. This does not make sense, and in fact mypy complains about
+# it. But pylance does not. Headscratcher. Have tried renaming file to `main.py`
+# and that did not solve it.
+from validators.directory_validation import (

```

Comment:
@bradbanister @lalrg please see the note above. Can you try pulling down this branch and running the script as is, and try running with the expected `from lms_file_tester.validators...` line? I'm quite perplexed about this.

To run from the `file-tester` directory: 

```
poetry run python lms_file_tester ../../docs/sample-out
```

Reply:
I get the same behavoir. 
Note: with python 3.9.

Reply:
Well, I'm able to suppress the mypy error anyway.